configuration/items.md

```diff
@@ -118,7 +118,20 @@ In the example above, if you move the Slider widget to 60%, move the Switch to O
 The Item name is the unique identified of the Item.
 The name should only consist of letters, numbers and the underscore character.
 Spaces and special characters cannot be used.
-A good naming schema can be advised.
+
+**Style Guide:**
+It is advised to follow a predefined naming scheme.
+The officially recommended scheme is used throughout this documentation and builds on a hierarchy idea.
+Floor or room names can be abbreviated to reduce overall name length.
+
+Every word of the Item name starts with a uppercase letter.
```

Comment:
Except for the device Group items apparently :-)
```
Every word of the Item name starts with a uppercase letter, unless it's a `Group` item.
```

Reply:
Reminder to self: "an uppercase"

Reply:
I don't understand. Besides the g everything still starts with an uppercase letter!?

Reply:
I agree with @ThomDietrich, using "unless" can get confusing, although if you want to highlight the difference adding "Group names are prepended with a lowercase 'g'" won't hurt right?

Reply:
I always do  Group_LivingRoom etc instead of GF_Living , when i scroll through logs/paperui its then easy to see whats Groups and whats not.. And simimliar Temperature_MasterBedRoom, Light_Kitchen_Island,  Light_Spot1